explainers/subresource-loading.md

```diff
@@ -156,6 +156,9 @@ subresources, `a.js`, `b.js`, and `c.png`, from the bundle.
 `urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6` is also loaded from
 the bundle, and a subframe is instantiated as an
 [opaque origin](https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque) frame.
+If a URL is available from an attached bundle, the browser must retrieve it from
+the bundle, instead of using any [registered custom protocolhandler](https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-registerprotocolhandler)
```

Comment:
```suggestion
the bundle, instead of using any [registered custom protocol handler](https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-registerprotocolhandler)
```

Reply:
Done.
Thank you.